src/main/kotlin/org/partiql/scribe/targets/trino/TrinoAstToSql.kt

```diff
@@ -290,4 +292,25 @@ public open class TrinoAstToSql(context: ScribeContext) : AstToSql(context) {
         t = t concat ")"
         return t
     }
+
+    private fun type(
+        symbol: String,
+        vararg args: Int?,
+        gap: Boolean = false,
+    ): SqlBlock {
+        val p = args.filterNotNull()
+        val t =
+            when {
+                p.isEmpty() -> symbol
+                else -> {
+                    val a = p.joinToString(",")
+                    when (gap) {
+                        true -> "$symbol ($a)"
+                        else -> "$symbol($a)"
+                    }
+                }
+            }
+        // types are modeled as text; as we don't want to reflow
+        return SqlBlock.Text(t)
+    }
```

Comment:
Could you please elaborate why we need this as a private fun? 

Why we cannot use the one from parent class?